.travis.yml

```
@@ -2,32 +2,9 @@ language: ruby
 sudo: false
 cache:
   - bundler
-before_install:
-  - gem install bundler
-
-# might add this back later
-# addons:
-#   apt:
-#     sources:
-#       - chef-current-trusty
-#     packages:
-#       - chefdk
-
-# Install bundle, to hell with saving a minute as it causes users to
-# search and realize that this is coming from some unpinned chefdk install
-install:
```

pretty sure we need this.

---

yup! i just deleted it since its the travis default (see https://docs.travis-ci.com/user/languages/ruby/) and they include optimizations like `--jobs=3 --retry=3`

---

I personally prefer explicit to implicit as defaults change and it makes it more resilient to changes. I am open to adding those additional options but to play devils advocate if say rubygems.org is down or being slow for any meaningful ammount of time a retry of 3 is not likely going to do anything other than increase the time it takes to fail. I personally prefer to fail fast in CI scenarios since uptime is not as important as in a long running application.